### PR TITLE
added radix package

### LINKED ISSUE #294: Add Radix
## Description

Add Radix chain support, supposed to go live on Maya mid-July. https://github.com/xchainjs/xchainjs-lib/tree/master/packages/xchain-radix

### PR DESCRIPTION
**Pull request by Thorian1te**

### CODE REVIEW

File: src/shared/const.ts [hunk 1]
```diff
@@ -14,11 +14,14 @@ import { DEFAULT_MIDGARD_MAYA_URLS } from './mayaMidgard/const'
 import { DEFAULT_MIDGARD_URLS } from './midgard/const'
 import { DEFAULT_THORNODE_API_URLS, DEFAULT_THORNODE_RPC_URLS } from './thorchain/const'
 import { DEFAULT_ENABLED_CHAINS, EnabledChain } from './utils/chain'
+import { envOrDefault } from './utils/env'
+
+require('dotenv').config()
+
+export const ASGARDEX_THORNAME = envOrDefault(process.env.REACT_APP_ASGARDEX_THORNAME, '')
```
Comment (cinnamoroll6130) [inline src/shared/const.ts]: No need to add `dx` as default value?

**Review by cinnamoroll6130 — COMMENTED** [review]
Need your input for the comment

**Review by cinnamoroll6130 — APPROVED** [review]
lgtm! 🚀

**Review by GrassRootsCrypto — APPROVED** [review]
Tested swap, add and withdraw - looks good.

**Pull request merged**